src/test/java/ru/stepev/test/training/at/hw1/CalculatorDivideTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.stepev.test.training.at.hw1;
+
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+import ru.stepev.test.training.at.hw1.dataprovider.DivideDataProvider;
+
+public class CalculatorDivideTest extends CalculatorTest {
+
+    public static final double DELTA = 0.01;
```

Comment:
I suppose that `DELTA` could be extracted to the `CalculatorTest` class as `protected` variable

Reply:
I've changed it.